tests/ui/issues/issue-4736.stderr

```diff
@@ -9,8 +9,8 @@ LL |     let z = NonCopyable{ p: () };
    |
 help: `NonCopyable` is a tuple struct, use the appropriate syntax
    |
-LL |     let z = NonCopyable(/* fields */);
-   |             ~~~~~~~~~~~~~~~~~~~~~~~~~
+LL |     let z = NonCopyable(/* () */);
```

Comment:
This one's a little confusing because `()` is the name of the type and its one value. But it's probably not worth specializing it to removes the comment markers.

Reply:
We already have a "Give us a default value for this type" logic. I do think that this and others are edge cases that we can ignore for now.